redbox/storage/elasticsearch.py

```diff
@@ -122,3 +125,15 @@ def list_all_items(self, model_type: str):
             return []
         uuids = [item["_id"] for item in results]
         return uuids
+
+    def get_file_chunks(self, parent_file_uuid: UUID) -> list[Chunk]:
+        """get chunks for a given file"""
+        target_index = f"{self.root_index}-chunk"
+
+        res = [
+            item["_source"]
+            for item in self.es_client.search(
+                index=target_index, query={"bool": {"filter": [{"term": {"parent_file_uuid": str(parent_file_uuid)}}]}}
```

Comment:
We should use the scan functionality shown further up to ensure proper retrieval of large results.

Reply:
that would explain why my test didn't pass!